Hunting Queries/BehaviorAnalytics/Anomalous Account Creation.yaml

```diff
@@ -0,0 +1,26 @@
+id: 0454c8d8-d0a6-42a4-8d03-f5b4fdcbd173
+name: Anomalous Account Creation

```

Comment:
We may want to consider a name that includes the type of account created, meaning Windows or Linux or Azure, in this case Azure AD.  The reason is that this specific table doesn't align with a source type being from AD or AAD.